### DISCUSSION TITLE
Exporting to bookmarks

### DISCUSSION THREAD

**Discussion by 4sb8** [fastaddons/GroupSpeedDial#303] [Q&A] [answered]
How can I export Group Speed Dial information into a bookmarks file. For my purposes FireFox would be ideal, but it could be any browser?

**Comment by Juraj-Masiar** [comment, marked as answer]
Hello,
I'm sorry for the huge delay, the notification email got "lost" :(

There is an experimental and "incomplete" backup that should be able to generate the "backup.html" file.
You can find it in the Options page / Import backup tab / "Export to HTML" (at the very bottom of that page).

**Reply by 4sb8** [reply]
Thank you - worked perfectly
